crates/occt-sys/build.rs

```diff
@@ -2,19 +2,31 @@ const LIB_DIR: &str = "lib";
 const INCLUDE_DIR: &str = "include";
 
 fn main() {
+    let current_dir = std::env::current_dir().expect("Should have a 'current' directory");
+    let patch_dir = current_dir.join("patch");
+
     let dst = cmake::Config::new("OCCT")
+        // TODO(bschwind) - Verify this is set properly
+        // .define("CMAKE_BUILD_TYPE", "Release")
```

Comment:
CMAKE_BUILD_TYPE should be handled correctly by `cmake` crate, ref. https://docs.rs/cmake/latest/cmake/struct.Config.html#method.profile

Reply:
Ah nice, I had hoped that was the case but hadn't checked yet.